### PR TITLE
feat: add top wallets page and fix eslints

### PR DESCRIPTION
**Pull request by Leon1777**
- add top wallets page 3412e7f4aea10fe83aa889ede682c2412dd3fa6f
  - data is fetched using the REST API URL stored in an environment variable, and the page displays a loading spinner until the data is ready
  - in css `.topwallets` set: overflow: visible otherwise the address is not fully visible
  - addressed some eslints warnings in App.js
- correct api address import and use https for fetch requests e143ff91a83d7471706058f5df439900840a0a61
- fix eslint warnings in multiple components de6f9e79cf7811b58934887251b69097b9efe5ac
- fix type mismatch in getOutputFromIndex function ed8258c23814309df3248abc2e8ff51cfe0b5a84
- higher resolution logo img 39f694b0541fb040827b005dc70ad8fb0f770ebc
##
- dynamic hashrate unit assignment 32a7a56a3ee6afee1d170864bcdb42a3a6b38079
##
- move api calls to karlsen-api-client.js 226e8d892753b941e1e85a821a0b480f2f1d0ee0
##
- add minimal-pie-chart 5e02af915c09e89b4088a59a243a7d1a7e1c1260
- bump version to `2.0.0`

### CODE REVIEW

**Comment by lemois-1337** [comment]
@Leon1777 This is amazing PR. I will merge it once we fixed the DB filler bug next week to have precise and correct real-time balances.

**Comment by Leon1777** [comment]
that should cover everything :)

File: src/components/TopWallets.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  }, []);
+
+  return (
+    <div className="blocks-page">
+      <Container className="webpage px-md-5 blocks-page-overview" fluid>
+        <div className="block-overview mb-4">
+          <div className="d-flex flex-row w-100">
+            <h4 className="block-overview-header text-center w-100 mt-4">
+              <FaWallet size="1.7rem" />
+              TOP WALLETS
```
Comment (lemois-1337) [inline src/components/TopWallets.js]: I might be a bit pedantic, but currently in the top menu we have all "single word" entries "Dashboard", "Blocks" and "Transactions". Could we please rename "Top Wallets" to something else using single word? Eg. "Tops" or "Distribution" or "Richlist" (didn't like that much) or "Wealthlist" or "Rank", maybe ChatGPT knows more. :)

Comment (Leon1777) [reply src/components/TopWallets.js]: `Distribution` sounds good

**Comment by lemois-1337** [comment]
Merged, looks great. The [Karlsen Analytics API](https://api.karlsencoin.com/docs#/Karlsen%20analytics) is fixed now and provides real-time address balances.

**Pull request merged**